docs/static/logging.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[logging]]
+== Logstash's Internal Logging
+
+Logstash emits internal logs during it's operation, which are placed in `LS_HOME`/logs. The default logging level is `INFO`. 
```

Comment:
typo: should be "its operation". Also, I think you might want the back ticks around the whole path `LS_HOME/logs`